packs/spore-war-bestiary/book-1-whispers-in-the-dirt/eshmok.json

```
@@ -27,11 +27,14 @@
                     "title": ""
                 },
                 "rules": [],
+                "showSlotlessLevels": {
+                    "value": false
+                },
                 "slots": {},
                 "slug": null,
                 "spelldc": {
                     "dc": 28,
-                    "value": 0
+                    "value": 18
```

```suggestion
                    "value": 20
```
NPC spell attack is DC - 8.